### DISCUSSION TITLE
Arabic Fine-tuning

### DISCUSSION THREAD

**Discussion by Hegelim** [PaddlePaddle/PaddleOCR#12211] [Ideas & Features]
## Question
1. which model is recommended for fine-tuning Arabic? I found 2 options: 
- https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/finetune_en.md#32-model, which is ch_PPOCRv3, suggested on the page, this is the best model for fine-tuning. However, I am not sure whether I should use this given that it starts with "ch" and I am assuming it is mainly used for Chinese
- https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/models_list_en.md#23-multilingual-recognition-modelupdating, here I found arabic_PP-OCRv3_rec. However, this model is trained (not pre-trained) and the size is too small, I am concerned whether it can be used for fine-tuning
Which one should I use?
2. Let's say I want to fine-tune the model on both English and Arabic and I am giving it a customized dictionary that has both English and Arabic letters. From what I see here https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/ppocr/postprocess/rec_postprocess.py#L42, whether the text is reversed or not is purely based on whether the word "arabic" is in the path of the dictionary? That sounds very strange. What should I do if the dictionary has both Arabic and English letters (not words)?

## 系统环境/System Environment：
Ubuntu 18.04

## 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
paddleocr 2.6.1.0
paddlepaddle-gpu 2.4.2

**Comment by Hegelim** [comment]
Could anyone help? @andyjpaddle @ToddBear

**Comment by Hegelim** [comment]
Now that PPOCRv4 models have been released, which pre-trained model is suggested to fine-tune Arabic? Could you please help @Topdu

**Comment by connorourke** [comment]
@Hegelim - have you made any progress with fine-tuning Arabic? 

I have a dataset the standard model doesnt handle so well, so I am keen to see if it can be improved (both detection & recognition).

Any advice would be useful!

**Comment by UserWangZz** [comment]
This issue has not been updated for a long time. This issue is temporarily closed and can be reopened if necessary.

**Comment by KyleNewman97** [comment]
The trainable PP-OCRv4 Arabic model can be found at:

https://paddleocr.bj.bcebos.com/PP-OCRv4/multilingual/arabic_PP-OCRv4_rec_train.tar

**Reply by Dawood180** [reply]
Hello, do you know the architecture used in this model?
When I was loading the weights , it gave me many warnings that the parameters arenot in the model.
Do you know the config file used to fine-tune this pre-trained model?
Any advice would be useful.

**Comment by lsuhail** [comment]
we need v5 arabic, i may contripute training it. if dataset avalible i will be thankful. if not i need guidece beased on your experance to achive highest results paddleOCR can achive higher arabic OCR bencmark score. which in return have great value.
